Dragon.cpp

```diff
@@ -1,11 +1,19 @@
 #include "Dragon.h"
 #include "DragonSlayer.h"
+#include "Character.h"
+#include "Utility.h"
 
-//Dragon::Dragon
+Dragon::Dragon
+(
+    std::string name_, 
+    int hitPoints_, 
+    int armor_
+) : Character(hitPoints_, armor_, 80), name(name_) {}
```

Comment:
Please never write functions that looks like this.

The following is 100x easier to read.
```
Dragon::Dragon( std::string name_, int hitPoints_,  int armor_ ) : 
Character(hitPoints_, armor_, 80), 
name(name_) 
{

}
```

Reply:
ok done.